docs/state.md

```diff
@@ -9,13 +9,13 @@ While Linux is usable on all T2 models, some features are limited by lack of dri
 - USB
 - Keyboard
 - Camera
+- Wifi
```

Comment:
Some models do not have support, notably the 15,4

Reply:
The ones without working wifi have the BCM4377 chip: MacBookPro15,4, MacBookPro16,3, MacBookAir9,1 and possibly MacBookAir8,2 (idk which chip that one has)

Reply:
I'd still rather have that information in the wifi guide

Reply:
Done the corrections